Whiskey/Tasks/Publish-WhiskeyBuildMasterPackage.ps1

```diff
@@ -168,17 +127,13 @@ function Publish-WhiskeyBuildMasterPackage
     $package = New-BMPackage -Session $buildMasterSession -Release $release -PackageNumber $packageName -Variable $variables -ErrorAction Stop
     $package | Format-List | Out-String | Write-Verbose
 
-    if( $skipDeploy )
+    if( $TaskParameter['SkipDeploy'] )

```

Comment:
All values converted from YAML are assumed to be strings, so this will always be true. You need to call `ConvertFrom-WhiskeyYamlScalar` to convert the string value to a boolean. Update your tests to pass in a string boolean value.